src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include "pal_types.h"
+
+#ifdef  GSSFW
```

Comment:
Another place where we assumed pal_ssl.h was doing the approved thing so we deliberately put them in the header file flying in the face of conventional wisdom :)